pages/Documents/DeveloperTools/LivePersonFunctions/ExternalInvocations/error-codes.md

```diff
@@ -27,3 +27,19 @@ This section describes possible errors that can occur when invoking a lambda fro
 | 'com.liveperson.faas.handler.log-limit-reached'     | Lambda exceeds log limits (see https://developers.liveperson.com/liveperson-functions-monitoring-logs.html)      |
 | 'com.liveperson.faas.fm.general'                    | General error, no specific reason                                                                               |
 | 'com.liveperson.faas.fm.unavailable'                | Could not reach/access remote system                                                                            |
+
+
+## Http codes
+
+| Code                                               | name             | Description                                              |
+|----------------------------------------------------|-----------------|-----------------------------------------------|
+| '200','202'           |  Processed/Accepted   | The request was processed successfully and the result of the request is included in the response.   |
+| '400'           |  Bad request  | Some validations have failed on the request                                             |
+| '401'           | Unauthorized    | Unauthorized,The request cannot be made without valid authentication                   |
```

Comment:
```suggestion
| '401', '403'          | Unauthorized    | Missing authentication or missing privileges               |
```

Reply:
Better to combine where it makes sense, as those error codes are nothing out of this world.

Reply:
updated